CodeGen/kubernetes/intel/cpu/xeon/manifest/codegen.yaml

```diff
@@ -1,31 +1,4 @@
 ---
-# Source: codegen/charts/codegen-ui/templates/configmap.yaml
-# Copyright (C) 2024 Intel Corporation
-# SPDX-License-Identifier: Apache-2.0
-
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: codegen-codegen-ui-config
-  labels:
-    helm.sh/chart: codegen-ui-1.0.0
-    app.kubernetes.io/name: codegen-ui
-    app.kubernetes.io/instance: codegen
-    app.kubernetes.io/version: "v1.0"
-    app.kubernetes.io/managed-by: Helm
-data:
-  APP_BACKEND_SERVICE_ENDPOINT: "/v1/codegen"
-  APP_DATA_PREP_SERVICE_URL: "/v1/dataprep"
-  CHAT_BASE_URL: "/v1/codegen"
-  UPLOAD_FILE_BASE_URL: "/v1/dataprep"
-  GET_FILE: "/v1/dataprep/get_file"
-  DELETE_FILE: "/v1/dataprep/delete_file"
-  BASE_URL: "/v1/codegen"
-  DOC_BASE_URL: "/v1/codegen"
-  BASIC_URL: "/v1/codegen"
```

Comment:
we can't remove `BASIC_URL` here. Because the default [ BASIC_URL  value](https://github.com/opea-project/GenAIExamples/blob/main/CodeGen/ui/svelte/.env) in codegen-ui doesn't work for K8S environment

Reply:
We can set it up separately for codeGen, without adding unnecessary variable configuration items to each Example. In fact, if you use svelte ui, it will be forwarded directly through the suffix, without adding config.